app/src/main/java/org/coepi/android/cen/CenRepo.kt

```diff
@@ -29,8 +30,11 @@ class CenRepo(private val cenApi: CENApi, private val cenDao: RealmCenDao, priva
 
     // last time (unix timestamp) the CENKeys were requested
     var lastCENKeysCheck = 0
+    private val TAG_LOG = "CenRepo"
+
 
     init {
+        Log.i(TAG_LOG, "init!")
```

Comment:
This is for you to debug locally, don't upload to repo

Reply:
well it could be useful to debug when it crashes, couldn't it?